site/src/pages/components/invokers.explainer.mdx

```diff
@@ -287,41 +287,43 @@ element types are handled.
-| `<dialog>`      | `'cancel'`            | If the `<dialog>` is `open`, cancel the dialog                                       |
-| `<select>`      | `'showPicker'`        | Call `.showPicker()` on the invokee                                                  |
-| `<input>`       | `'showPicker'`        | Call `.showPicker()` on the invokee                                                  |
-| `<video>`       | `'playpause'`         | Toggle the `.playing` value                                                          |
-| `<video>`       | `'pause'`             | If `.playing` is `true`, set it to `false`                                           |
-| `<video>`       | `'play'`              | If `.playing` is `false`, set it to `true`                                           |
-| `<video>`       | `'toggleMuted'`       | Toggle the `.muted` value                                                            |
-| `<audio>`       | `'playpause'`         | Toggle the `.playing` value                                                          |
-| `<audio>`       | `'pause'`             | If `.playing` is `true`, set it to `false`                                           |
-| `<audio>`       | `'play'`              | If `.playing` is `false`, set it to `true`                                           |
-| `<audio>`       | `'toggleMuted'`       | Toggle the `.muted` value                                                            |
-| `<*>`           | `'toggleFullscreen'`  | If the element is fullscreen, then exit, otherwise request to enter                  |
-| `<*>`           | `'requestFullscreen'` | Request the element to enter into 'fullscreen' mode                                  |
-| `<*>`           | `'exitFullscreen'`    | Request the element to exit 'fullscreen' mode                                        |
-| `<input type=number>` | `'stepUp'`      | Call `.stepUp()` on the invokee                                                      |
-| `<input type=number>` | `'stepDown'`    | Call `.stepDown()` on the invokee                                                    |
-
-> Note: The above tables are an attempt at wide coverage, but ideas are welcome. Please submit a PR if you have one!
+| Invokee Element | `action` hint         | Behaviour                                                                                                  |
+| :-------------- | :-------------------- | :--------------------------------------------------------------------------------------------------------- |
+| `<* popover>`   | `'toggle-popover'`    | Shows the `popover` if closed, otherwise hides. Similar to `.togglePopover()`                              |
+| `<* popover>`   | `'hide-popover'`      | Hides the `popover` if open, otherwise does nothing. Similar to `.hidePopover()`                           |
```

Comment:
Ahh, I see where the "starts with `-`" came from. Perhaps the custom ones should use a starting double-dash `--` to match CSS? I'd just hate to mint yet another naming scheme. I'm surprised @annevk was supportive of this naming scheme. Perhaps it exists somewhere else in HTML that I'm not thinking of?

Reply:
I agree, this needs some more thought. Hopefully we'll discuss in the openui call in ~10hrs 😆